solutions/swb-reference/src/templates/README.md

```diff
@@ -1,30 +1,48 @@
+
+## Onboard Account TGW
+This [template](./onboard-account-tgw.cfn.yaml) onboards a hosting account as well as helps integrate with the resources created as part of the `new-firewall-vpc.yaml` template deployed in the Central Network account. You need to provide the Gateway Load Balancer Service ID and Transit Gateway ID output values from the Central Network account to this template's input parameters. This template provisions network components (VPC and Subnets) based on the public/private nature selected during stack creation.
 
 ### Dependencies
-As is, these templates depend on a subscription to an EC2 firewall appliance. We recommend you or your SA review the network settings in both the `new-firewall-vpc.yaml` file and the `onboard-account-tgw.cfn.yaml` to update them to best suit your needs. This may include removing the EC2 firewall and the Gatewayload balancer, and updating the Transit Gateway's and onboarding account's route tables to reroute trafic that used to go through the EC2 firewall.
+As is, these templates depend on a subscription to an EC2 firewall appliance. We recommend you or your SA review the network settings in both the `new-firewall-vpc.yaml` file and the `onboard-account-tgw.cfn.yaml` to update them to best suit your needs.
 
 ### How to use these:
-1. Launch the stack in a network account
+1. Launch the stack in a central network account
 1. Share the network account's transit gateway with the hosting account
-    1.Go to `VPC` -> `Transit Gateways` -> `Transit Gateways`
+    1. Go to `VPC` -> `Transit Gateways` -> `Transit Gateways`
     1. Select the transit gateway and, from the Actions menu, select "Share transit gateway" 
     1. If a resource share doesn't already exist for your TGW, then create one. Make sure to give access to the app account on the "Grant access to principals"
-1. On the onboarding account, accept the resource share
+1. On the hosting account, accept the resource share
     1. Resource Access Manager -> Shared with me -> Accept the invite
-1. Launch the onboarding stack (`onboard-account-tgw.cfn.yaml`) in the hosting account with the relevant outputs from the network account onboarding
-1. Use the outputs from the onboarding stack to onboard the account with the main account per the instructions in the [setup instructions](../../README.md#setup-instructions-for-swbv2p1).
+1. Launch the onboarding stack (`onboard-account-tgw.cfn.yaml`) in the hosting account with the relevant outputs from the central network account stack.
+1. Since this stack creates its own network components, associate its subnets to Transit Gateway by adding them to the SecurityVPC route table. This helps centrally inspect egress traffic inspection.
+1. Create Gateway Load Balancer endpoints in the hosting account. Each endpoint needs a subnet per AZ. The Internet Gateway route table needs to have these endpoints added for ingress traffic inspection. (Ignore this step if using private hosting account subnets)
```

Comment:
Availability Zone instead of AZ?

Reply:
I would move the `ignore this step` condition to the front. Maybe just say "If you are using public hosting account subnets, ..."